From 3d2c72905a8ea2ff38bf36945a271014f646396a Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Wed, 1 Jun 2016 14:08:41 +0800
Subject: [PATCH] arm64: configs: update rockchip config by savedefconfig

After mali driver upgrade, there are not MALI_MIDGARD_DEBUG_SYS
any more.

Change-Id: I8ecefb576b91faf9c366bf5dd3e2b4d4613d654c
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 1 -
 arch/arm64/configs/rockchip_defconfig      | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index c84733fe794a..584a12d04e09 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -362,7 +362,6 @@ CONFIG_ROCKCHIP_DW_MIPI_DSI=y
 CONFIG_ROCKCHIP_ANALOGIX_DP=y
 CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_MALI_MIDGARD=y
-CONFIG_MALI_MIDGARD_DEBUG_SYS=y
 CONFIG_MALI_DEVFREQ=y
 CONFIG_MALI_EXPERT=y
 CONFIG_MALI_DEBUG_SHADER_SPLIT_FS=y
diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 79f7c1aa0ece..b292450c684f 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -352,7 +352,6 @@ CONFIG_ROCKCHIP_DW_HDMI=y
 CONFIG_ROCKCHIP_DW_MIPI_DSI=y
 CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_MALI_MIDGARD=y
-CONFIG_MALI_MIDGARD_DEBUG_SYS=y
 CONFIG_MALI_DEVFREQ=y
 CONFIG_MALI_EXPERT=y
 CONFIG_MALI_DEBUG_SHADER_SPLIT_FS=y
-- 
2.35.3

